lib/features/reps_core/reps_admin/reps_admin.module

```diff
@@ -413,3 +413,22 @@ function reps_admin_theme_registry_alter(&$theme_registry) {
     }
   }
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function reps_admin_form_alter(&$form, &$form_state, $form_id) {
+  // views_form_core_content_administration_page.
+  if ($form_id == 'views_exposed_form') {
+    $options = &$form['type']['#options'];
+    if (!empty($options)) {
+      $node_entities = node_type_get_types();
+      foreach ($node_entities as $node_entity) {
```

Comment:
minor: call the function directly in the loop 
`foreach (node_type_get_types() as $node_entity) {`

but instead of an extra db call that has an impact on the performance couldn't you use values from $options?
`foreach ($options as $ct_name => $title)` ?